app/src/main/java/com/anytypeio/anytype/ui/editor/sheets/ObjectMenuBaseFragment.kt

```diff
@@ -354,7 +354,6 @@ abstract class ObjectMenuBaseFragment :
     }
 
     interface DocumentMenuActionReceiver {
-        fun onMoveToBinSuccess()
```

Comment:
idem

Reply:
we don't need it anymore, navigation is used in that case